src/components/postgresConnect.ts

```diff
@@ -107,11 +109,11 @@ class PostgresDB {
     }
   }
 
-  public getActionsQueued = async (
+  public getActionsScheduled = async (
     payload: ActionQueueGetPayload = { status: 'Scheduled' }
   ): Promise<ActionQueue[]> => {
     const text =
-      'SELECT id, action_code, parameters, execution_time FROM action_queue WHERE status = $1 ORDER BY execution_time'
+      'SELECT id, action_code, trigger_payload, parameter_queries, time_completed FROM action_queue WHERE status = $1 ORDER BY time_completed'
```

Comment:
Hmm, this one is looking for actions that are scheduled for the future ? If so, kind of weird to have 'time_completed' in that case

Reply:
Yeah, fair point. I originally had this field called `execution_time`, but I didn't like the inconsistent naming with `time_queued`. I guess we could have an additional column called `time_scheduled` (and just leave it `null` for non-scheduled actions), but let's discuss it before I re-address Scheduled actions (they're somewhat broken as a result of these changes -- have added a clarifying note to the original post here).

Reply:
i like that idea, 'scheduled_time or time_scheduled`

Reply:
Okay, I have added a `time_scheduled` column. However, I have NOT changed the scheduled actions to use this as they're currently broken and beyond this issue and current requirements. (So that column is just a placeholder for now.)